### PR TITLE
Adds a new sample events page and adds it to Percy

### PR DESCRIPTION
**Pull request by johnrich85**
Fixes: #4615 

Changes proposed in this pull request:

- Adds a new sample events page
- Tells Percy to snapshot it

### CODE REVIEW

**Comment by devnook** [comment]
@johnrich85 What's the status?

**Review by matthiasrohmer — APPROVED** [review]

File: site/_js/web-components/web-tabs.js [hunk 1]
```diff
@@ -111,12 +111,16 @@ export class WebTabs extends BaseElement {
   }
 
   render() {
-    this._tabPanels = Array.from(this.children);
-    const tabs = this._formatTabs();
-    return html`
-      <div role="tablist">${tabs}</div>
-      ${this._tabPanels}
-    `;
+    if (this.hasAttribute('data-unresolved')) {
```
Comment (matthiasrohmer) [inline site/_js/web-components/web-tabs.js:114]: Nit: prefer early exit, so `if (this.dataset.resolved)`

File: site/_js/web-components/web-tabs.js [hunk 2]
```diff
@@ -111,12 +111,16 @@ export class WebTabs extends BaseElement {
...
+      this._tabPanels = Array.from(this.children);
+      const tabs = this._formatTabs();
+      return html`
+        <div role="tablist">${tabs}</div>
+        ${this._tabPanels}
+      `;
+    }
+
+    return [Array.from(this.children)];
```
Comment (matthiasrohmer) [inline site/_js/web-components/web-tabs.js:123]: Is the wrapping array intentional?

**Review by matthiasrohmer — COMMENTED** [review]
Looks almost good to me. Further thinking about it, I think it would be more intuitive to add a `data-resolved` attribute on first render, instead of the other way round, no? That would also save you from going over all existing usages of `web-tab`

**Review by matthiasrohmer — APPROVED** [review]
Can you try resetting the unrelated Markdown whitespace changes? Otherwise LGTM, thanks! 🙂

**Comment by johnrich85** [comment]
Hey @rachelandrew , would you mind giving the content a quick once over, please?

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**